compute/k8s-services/main.tf

```diff
@@ -247,6 +248,46 @@ module "traefik_alb_anon_dns_core_alias" {
 }
 
 
+# --------------------------------------------------
+# Traefik v2: For validation and testing
+# --------------------------------------------------
+
+module "traefikv2_alb_anon" {
+  source                = "../../_sub/compute/eks-alb"
+  deploy                = var.traefikv2_test_alb_deploy
```

Comment:
Do we want the word "test" in the variable name?

Reply:
I'm honestly not sure.

To give a bit of background the intention is that this variable will deploy some ALB's that will only be used for testing the new Traefik v2 implementation.  Once it's tested and confirmed as working then they will be torn down (by changing the variable to false) and then the existing ALB's will be updated.

We could perhaps use traefikv2_migration_alb_deploy instead possibly but i'm not convinced the name conveys their real purpose.

Reply:
Ok, then I'm happy with whatever you go with

Reply:
We will stick with test for now, but only on the basis that once we're done with the testing phase then the code will most likely be stripped from the module anyway.  Rasmus had originally said to me about possibly doing the code outside of infrastructure-modules, but doing so would have been much more trouble because of the dependencies.